app/views/user/swaps/_polls_advice_self.html.haml

```diff
@@ -0,0 +1,29 @@
+%p
```

Comment:
Please can we rename these files so that they don't contain the word "advice" :stuck_out_tongue_winking_eye:

I suggest "interpretation".

Reply:
Done